src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age !== undefined && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried
+        ? person.sex === 'f'
+          ? `${person.partnerName} is my husband`
+          : `${person.partnerName} is my wife`
+        : 'I am not married'}
```

Comment:
using nested ternary operator is a bad practice, it's better to move it to the variable